Packs/Yara/Scripts/YaraScan/YaraScan.py

```diff
@@ -42,12 +42,12 @@ def main():
                 "HasMatch": False,
                 "HasError": False,
                 "MatchCount": 0,
-                "Matches": list(),
-                "Errors": list()
+                "Matches": [],
+                "Errors": []
             }
             try:
-                cRule = yara.compile(source=yaraRuleRaw)
```

Comment:
Would this not break backwards compatibility?

Reply:
Yeah it would, because the yara module requires a dictionary instead of a string to handle multiple yara rules at once, as mentioned here https://yara.readthedocs.io/en/stable/yarapython.html. To keep backwards compatibility it would require bigger changes to the automation itself to handle both cases, where input is either a single or multiple yara rules. How would you prefer to going forward with this?

Reply:
Breaking backwards compatibility is not an option but bigger changes are welcome. Maybe you can add a new argument?

Reply:
@jlevypaloalto in this case then I would also need to change the yaraRule argument to optional, because it doesn't make sense to also provide a single rule if you want to provide multiple rules to begin with. I believe this option is optimal as it wouldn't break backwards compatibility. However there is the case where someone can run the automation without providing any rules, as both yaraRule and yaraRules will be optional arguments. Then in that case how would you prefer to handle this case?

Reply:
@ioannispanagopoulos the implementation is up to you. I'd prefer if you make the argument support both a single and multiple rules.

Reply:
@jlevypaloalto Updated all files with the suggestions made and made it so the automation uses the yaraRuleRaw argument as it is with the added functionality for multiple rules. Let me know if there is any other change needed for this.